src/main/containers/Panels.jsx

```diff
@@ -21,19 +21,27 @@ const Panels = () => {
 
   return (
     <PanelsWrapper>
-      <SourcePanel treeCount={treeCount}
-                   updateTreeCount={setTreeCount}
-                   data={data}
-                   setData={setData}
-                   setTests={setTests} />
-      <TestPanel treeCount={treeCount}
-                 updateTreeCount={setTreeCount}
-                 data={data}
-                 setTests={setTests}
-                 tests={tests} />
-      <DestinationPanel
-                 tests={tests}
-                 setTests={setTests} />
+      <SourcePanel
+        onClickFunction={() => setActivePanel('source')}
+        treeCount={treeCount}
+        updateTreeCount={setTreeCount}
+        data={data}
+        setData={setData}
+        setTests={setTests}
+        active={(activePanel === 'source')} />
+      <TestPanel 
+        onClickFunction={() => setActivePanel('test')}
+        treeCount={treeCount}
+        updateTreeCount={setTreeCount}
+        data={data}
+        setTests={setTests}
+        tests={tests}
+        active={(activePanel === 'test')} />
+      <DestinationPanel 
+        onClickFunction={() => setActivePanel('dest')}
```

Comment:
In a future PR, we could likely parmatize this function so it’s DRY-er but this works great for now!